### PR TITLE
#1073 Show place_holder_text as empty table placeholder in FlexibleTable

### LINKED ISSUE #1073: Support empty state for data tables in elyra/canvas
**Current empty table -**
current-empty-state

**New empty table -** 
Screen Shot 2022-06-06 at 4 18 02 PM

- Use [place_holder_text](https://github.com/elyra-ai/canvas/wiki/3.2-Common-Properties--UI-Hints#resource-definition) for the text used.
- If `place_holder_text` is not defined, leave it blank
- Empty table height 120px excluding header
- Empty table text typography token is `$body-long-01`

### PR DESCRIPTION
**Pull request by nmgokhale**
Fixes #1073 

For empty tables, header checkbox is unchecked - 
Screen Shot 2022-06-06 at 4 22 50 PM

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Review by matthoward366 — COMMENTED** [review]
Looks good.  1 minor comment

File: canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.scss [hunk 1]
```diff
@@ -128,3 +128,23 @@ $row-left-padding: $spacing-02;
...
+	@include carbon--type-style("body-long-01");
+}
+
+.properties-light-enabled {
+	.properties-ft-empty-table {
+		background-color: $ui-background;
+	}
+}
+
+.properties-light-disabled {
```
[the remark this answers is not in the document]
Comment (nmgokhale) [reply canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.scss:146]: To test this, there's a config option in common-properties section -
Screen Shot 2022-06-07 at 10 11 33 AM

[the remark this answers is not in the document]
Comment (matthoward366) [reply canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.scss:146]: I was thinking about testing the disabled part as well.

[the remark this answers is not in the document]
Comment (nmgokhale) [reply canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.scss:146]: **Structurelisteditor -**
Screen Shot 2022-06-07 at 3 55 53 PM

**Structuretable -**
Screen Shot 2022-06-07 at 3 56 35 PM
Under conditions tab -
Screen Shot 2022-06-07 at 3 56 49 PM

**GLMM -**
Screen Shot 2022-06-07 at 3 58 27 PM

**Review by matthoward366 — APPROVED** [review]
Looks good!

**Pull request merged**